pageComponents/NewPostPage/PlayerActionSelectors.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from "react";
+import styled from "styled-components";
+import StreetActionSelector from "@@/components/StreetActionSelector";
+import { MOBILE_MEDIA } from "@@/constants/mediaquery";
+import { HandAction, HandActionType, HandStreet } from "@@/models/Hand";
+import getPositionByPlayerAndIndex from "@@/utilities/getPositionByPlayerAndIndex";
+
+interface Props extends React.Attributes {
+  playerLength: number;
+  street: HandStreet;
+  playerActions: HandAction[];
+  className?: string;
+  style?: React.CSSProperties;
+  onChange?: (index: number, type: HandActionType, betSize: number) => void;
+}
+
+export default function PlayerActionSelectors({
+  playerLength,
+  street,
+  playerActions,
+  onChange = () => undefined,
+  ...props
+}: Props) {
+  let _tableMaxBetSize = 0;
+  const previousBetSizeMap = new Map(
+    Array.from({ length: playerLength }, (_, playerIndex) => [
+      playerIndex,
+      street === HandStreet.preflop && playerIndex === 1 ? 1 : 0,
+    ])
+  );
+
+  return (
+    <Root {...props}>
+      {playerActions.map(({ playerIndex, betSize }, index) => {
+        const tempTableMaxBetSize = _tableMaxBetSize;
+        _tableMaxBetSize = betSize;
+        const tempPreviousBetSize = previousBetSizeMap.get(playerIndex);
+        previousBetSizeMap.set(playerIndex, betSize);
```

Comment:
Here can have some room to improve. At least it's not so great to name variables `tempXxx`.